deployment/k8s/mysql-deployment.yaml

```diff
@@ -124,10 +124,22 @@ data:
     PREPARE stmt FROM @sql;
     EXECUTE stmt;
 
+    create table if not exists authenticator (
```

Comment:
Please consider to use password salting to store passwords in the DB table, otherwise it is not sufficiently secure.